clients/vault/src/system.rs

```diff
@@ -452,11 +452,8 @@ impl VaultService {
 		let is_public_network = wallet.is_public_network();
 
 		// re-submit transactions in the cache
-		let (_, errors) = wallet.resubmit_transactions_from_cache().await;
-		if !errors.is_empty() {
-			// todo: handle timeouts
-			tracing::error!("Failed to resubmit: {:?}", errors);
-		}
+		let _receivers = wallet.resubmit_transactions_from_cache().await;
+		//todo: handle errors from the receivers
```

Comment:
I think that we already able to remove this TODO comment because we have a pretty good logging mechanism inside `resubmit_transactions_from_cache` function

Reply:
it's for https://github.com/pendulum-chain/spacewalk/issues/340